ui/components/multichain/nft-item/index.scss

```diff
@@ -19,7 +23,33 @@
 
   &__item-image {
     border-radius: 8px;
-    padding: 0;
+    width: 100%;
+    object-fit: cover;
+    aspect-ratio: 1;
+  }
+
+  &__item-image--hidden {
+    border-radius: 8px;
+    width: 100%;
+    object-fit: cover;
+    aspect-ratio: 1;
+    background: rgba(255, 255, 255, 0.5);
+    mix-blend-mode: lighten;
+  }
+
+
+
+  @include design-system.screen-sm-max {
+    &__item-image--hidden {
+      background: rgba(255, 255, 255, 0.5);
+      mix-blend-mode: lighten;
```

Comment:
The Figma has a nice blur effect, but I think this is more appropriate (so we don't need to render images if they are hidden)

Is there an existing background color palette we can use for both light & dark mode?

Reply:
I will follow up with design team about this. Fwiw, this background does align well with the existing "show nft media" overlay that we use:

<img width="1188" alt="Screenshot 2025-01-10 at 2 09 32 PM" src="https://github.com/user-attachments/assets/a58a629b-a86a-44a4-9ea6-03336205ac47" />

I would personally prefer to handle this separately if you're cool with that. It might need some extra design work.

Reply:
Awesome, yep let's handle this as a separate task